html/captive-portal/lib/captiveportal/PacketFence/Controller/Node/Manager.pm

```diff
@@ -31,7 +31,7 @@ sub unreg :Local :Args(1) {
     my $node = node_view($mac);
     if ($username && $node) {
         $c->log->info("$username attempting to unregister $mac");
-        if ($username ne $default_pid && $username eq $node->{pid}) {
+        if (($username ne $default_pid || $username ne $admin_pid ) && $username eq $node->{pid}) {
```

Comment:
Just to simplify ($username ne $default_pid || $username ne $admin_pid ) = 1 and i suppose that it´s wrong, it should probably be ($username ne $default_pid && $username ne $admin_pid ) instead